gdal2mbtiles/helpers.py

```diff
@@ -32,7 +32,7 @@
 def image_mbtiles(inputfile, outputfile, metadata,
                   min_resolution=None, max_resolution=None, fill_borders=None,
                   zoom_offset=None, colors=None, renderer=None,
-                  preprocessor=None):
+                  preprocessor=None,pngdata={}):
```

Comment:
@vijaymadhavan, could you replace dict objects in the function definitions which an `if pngdata is None: pngdata = {}` inside the function blocks? The reason for this is that sometimes declaring a list or dict object in the kwargs can cause the data to leak into other calls of that function.